utils/misc.mli

```diff
@@ -116,6 +116,29 @@ module Stdlib : sig
     (** [split_at n l] returns the pair [before, after] where [before] is
         the [n] first elements of [l] and [after] the remaining ones.
         If [l] has less than [n] elements, raises Invalid_argument. *)
+
+    (** Returns [true] iff the given list, with respect to the given equality
```

Comment:
The style used in this module is to have docstrings after definitions.  It would seem better to stick to it.